code/modules/antagonists/eldritch_cult/eldritch_effects.dm

```diff
@@ -0,0 +1,213 @@
+		current_eldritch_knowledge.cleanup_atoms(selected_atoms)
+		return
+	to_chat(user,"<span class='warning'>Your ritual failed! You used either wrong components or are missing something important!</span>")
+
+/obj/effect/eldritch/big
+	name = "Transmutation rune"
+	icon = 'icons/effects/96x96.dmi'
+	icon_state = "eldritch_rune1"
+	pixel_x = -32 //So the big ol' 96x96 sprite shows up right
+	pixel_y = -32
+
+/**
+  * #Reality smash tracker
+  *
+  * Stupid fucking list holder, DONT create new ones, it will break the game, this is automnatically created whenever eldritch cultists are created.
+  *
+  * Tracks relevant data, generates relevant data, useful tool
+  */
+/datum/reality_smash_tracker
+	///list of tracked reality smashes
+	var/list/smashes = list()
+	///List of mobs with ability to see the smashes
+	var/list/targets = list()
+
+/datum/reality_smash_tracker/Destroy(force, ...)
+	if(GLOB.reality_smash_track == src)
+		stack_trace("/datum/reality_smash_tracker was deleted. Heretics may no longer access any influences. Fix it or call coder support")
+	for(var/X in smashes)
+		qdel(X)
+	for(var/Y in targets)
+		targets -= Y
+	return ..()
+
+/**
+  * Automatically fixes the target and smash network
+  *
+  * Fixes any bugs that are caused by late Generate() or exchanging clients
+  */
+/datum/reality_smash_tracker/proc/ReworkNetwork()
+	for(var/mind in targets)
+		for(var/X in smashes)
+			var/obj/effect/reality_smash/reality_smash = X
+			reality_smash.AddMind(mind)
+
+/**
+  * Generates a set amount of reality smashes based on the N value
+  *
+  * Automatically creates more reality smashes
+  */
+/datum/reality_smash_tracker/proc/Generate(var/n)
+	var/number = round(max(n * 6,10),1)
+
+	for(var/i in 0 to number)
+
+		var/obj/effect/landmark/L = pick(GLOB.generic_event_spawns + GLOB.landmarks_list)
+		var/obj/effect/reality_smash/RS = new(L.drop_location())
+		smashes += RS
+
+	ReworkNetwork()
+
+/**
+  * Adds a mind to the list of people that can see the reality smashes
+  *
+  * Use this whenever you want to add someone to the list
+  */
+/datum/reality_smash_tracker/proc/AddMind(var/datum/mind/M)
+	RegisterSignal(M.current,COMSIG_MOB_LOGIN,.proc/ReworkNetwork)
+	targets |= M
+	for(var/X in smashes)
+		var/obj/effect/reality_smash/reality_smash = X
+		reality_smash.AddMind(M)
+
+/**
+  * Removes a mind from the list of people that can see the reality smashes
+  *
+  * Use this whenever you want to remove someone from the list
+  */
+/datum/reality_smash_tracker/proc/RemoveMind(var/datum/mind/M)
+	targets -= M
+	for(var/obj/effect/reality_smash/RS in smashes)
+		RS.RemoveMind(M)
+
+/obj/effect/broken_illusion
+	name = "Pierced reality"
+	icon = 'icons/effects/eldritch.dmi'
+	icon_state = "pierced_illusion"
+
+/obj/effect/broken_illusion/examine(mob/user)
+	if(!IS_HERETIC(user) && ishuman(user))
+		var/mob/living/carbon/human/human_user = user
+		to_chat(human_user,"<span class='warning'>Your brain hurts when you look at this!</span>")
```

Comment:
you might wanna add a special tk_act() interaction for this (can effects even have that proc?), like the ones for the SM crystal and the singularity

Reply:
ooh, and maybe one for using the "look up" verb while standing directly underneath one of these (with no floor or other obstruction between you and the piercing in reality)